pkg/utils/yaml.go

```diff
@@ -102,6 +102,11 @@ func ScanIacDocumentsFromYaml(scanner *bufio.Scanner, byteArray []byte, filePath
 
 	dec := yaml.NewDecoder(bytes.NewReader(byteArray))
 
+	if dec == nil {
+		err := fmt.Errorf("error cretaing yaml decoder in ScanIacDocumentsFromYaml()")
```

Comment:
hey @jlk , I think the user wouldn't care to know the function name here. so we can keep the error message generic and simple. also, creating is mispelled

Reply:
requsting this change